src/components/MoneyReportHeader.js

```diff
@@ -57,6 +57,7 @@ const defaultProps = {
     session: {
         email: null,
     },
+    policy: undefined,
```

Comment:
@azamuddin Most of the places we have used {} as the default value. Should we do the same here?
```suggestion
    policy: {},
```

Reply:
Although the final function that consume policy is using `undefined` as default value, I have tested that everything works using `{}`, so yes for consistency I agree we can use `{}` instead of `undefined`